docs/core/integration-quality-scale/rules/iqs034.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "IQS034 - Dependency transparency"
+---
+
+IQS034 is as follows:
+
+Dependency transparency
+
+## Reasoning
+
+Home Assistant uses a lot of dependencies to work.
+These dependencies will be shipped with new versions of Home Assistant.
+In order for the project to trust the dependencies, we have a set of requirements we want the dependencies to meet.
+
+- The source code of the dependency must be available under an OSI-approved license.
+- The dependency must be available on PyPI.
+- The package published to PyPi should be built and published inside a CI pipeline.
+
+## Additional resources
+
+
+## Exceptions
+
+There are no exceptions to this rule.
```

Comment:
Can we enforce this rule on every library? I can imagine that there are some libraries that are maintained by someone who can just look at the code for 10 minutes every week and just publishes because that's what they have time for instead of reworking the whole CI

Reply:
I think the bar is too high if we want this rule to be for Bronze.

Reply:
I'm happy with the this rule being on bronze and the requirements around this.

Reasons is mostly around licensing and transparency around the supply chain and publications. Soon we'll be able to check this chain, and further down the line we will be able to verify signatures.

Honestly, I think this describes a bare minimum bar.

Reply:
Ok